client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -68,6 +86,58 @@ export default React.createClass( {
 		return ( this.props.domains || [ this.props.domain ] ).filter( domain => domain.currentUserCanManage );
 	},
 
+	wrongNSMappedDomains() {
+		debug( 'Rendering wrongNSMappedDomains' );
+		const wrongMappedDomains = this.getDomains().filter( domain =>
+			domain.type === domainTypes.MAPPED && ! domain.pointsToWpcom );
+
+		debug( 'Domains:', this.getDomains() );
```

Comment:
Oh, and my main point here (probably lost in my rant) was that this info (the list of all the domains) is not specific to this one warning, but to all of them, or even to the the whole domains list - which is where I'd prefer to have it. Then we can benefit from it during debuging of all the different warnings, not only this exact one.